DuckDuckGo/PrivacyProtection.storyboard

```diff
@@ -1,11 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="14313.18" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES" initialViewController="rnY-b4-XPk">
-    <device id="retina5_5" orientation="portrait">
-        <adaptation id="fullscreen"/>
-    </device>
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="14854.2" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES" initialViewController="5UB-cE-Uy8">
+    <device id="retina5_5" orientation="portrait" appearance="light"/>
```

Comment:
Navigation Bar in this storyboard should not be translucent, otherwise there is a slight color difference between it and the omni bar.

Reply:
There is also a glitch that does not appear on the version from develop:
1. Fresh install the app when the network is on (to fail downloading tracker lists).
2. Navigate to any page (it will fail - no network).
3. Open Privacy dashboard.

<img width="504" alt="glitch" src="https://user-images.githubusercontent.com/2865434/63155058-2d17f580-c012-11e9-8735-1f3f3e86bb3c.png">